R/utils.R

```diff
@@ -47,3 +47,20 @@ read_shape_zip <- function(path,layer){
 }
 
 
+read_gauge_googlesheets <- function(url=Sys.getenv("GFH_GAUGE_URL")){
+  sns <- sheet_names(ss = url)
+  sn_filt <- sns[sns!="Sheet1"]
```

Comment:
We only need to read in the gauge sheets I think, so also excluding `return_period` and `metadata`?

Reply:
we should keep metadata as it is the only tab with spatial information which we use. True that loading the `return period` tab is redundant, but I'd like to have one target at least in these early stages that just loads the entire workbook and we an filter from there. Having the RP data.frame could even end up saving a wrangling step so I don't think it's to negative to leave in at this stage